spikingjelly/activation_based/neuron/flexsn.py

```
@@ -274,15 +587,192 @@ def __init__(
+                build_training_kernels = None
+                attach_flexsn_handle_finalizer = None
+                register_flexsn_kernel_handle = None
+            if build_inference_kernel is not None:
+                try:
+                    self._inductor_scan_kernel, self._inductor_scan_info = (
+                        build_inference_kernel(core, num_inputs, num_states, num_outputs, example_inputs=example_inputs)
+                    )
+                    self._inductor_scan_final_state_kernel, self._inductor_scan_final_state_info = (
+                        build_inference_final_state_kernel(core, num_inputs, num_states, num_outputs, example_inputs=example_inputs)
+                    )
+                except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)